### PR TITLE
OSD-21000 - Enable KAS monitoring using internal routes

### PR DESCRIPTION
**Pull request by tnierman**
https://issues.redhat.com/browse/OSD-21000

---

Modifies the routemonitor logic to enable monitoring of kube-apiservers in hypershift using internal routes

### CODE REVIEW

File: pkg/blackboxexporter/blackboxexporter.go [hunk 1]
```diff
@@ -121,6 +121,25 @@ func (b *BlackBoxExporter) EnsureBlackBoxExporterServiceExists() error {
 	return nil
 }
 
+func (b *BlackBoxExporter) EnsureBlackBoxExporterConfigMapExists() error {
+	resource := corev1.ConfigMap{}
+	populationFunc := func() corev1.ConfigMap { return templateForBlackBoxExporterConfigMap(b.NamespacedName) }
```
Comment (tnierman) [inline pkg/blackboxexporter/blackboxexporter.go:126]: commenting on this function is verbose, but matches the style of the code in this file

**Review by mjlshen — APPROVED** [review]
/approve

**Pull request merged**